README.md

```diff
@@ -70,11 +71,11 @@ More details in [Wiki](https://github.com/IATkachenko/HA-SleepAsAndroid/wiki/app
     6. press `import` button
  2. create automatization based on blueprint:
     * `person` and `state` is using to run actions only if **person** in **state** (to avoid run home automatization related to sleep tracking while vacation in Siberia, for example)
-    * add actions for [evens]((https://docs.sleep.urbandroid.org/services/automation.html#events))
+    * add actions for [events]((https://docs.sleep.urbandroid.org/services/automation.html#events))
     
 ### on device event
  1. select `Device` in automatization trigger and use `SleepAsAndroid` device;
- 1. select trigger from a list.
+ 2. select trigger from a list.
```

Comment:
We don't need this: this is just md markup list syntax.

Reply:
was following suit because previous lists in the readme are actually sequential in the source doc.  I can drop it if you prefer, just seemed best to do the same thing everywhere.

Reply:
"Application configuration" section and "on device event" is using md-markup style, but "blueprint" is not.
I prefer to keep md-markup and fix blueprint in future.